hooks/persistence-azure-monitor/tests/__snapshot__/persistence-azure-monitor_test.yaml.snap

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
 matches the snapshot:
```

Comment:
This one fails the tests because the `affinity` and `tolerations` are missing, not sure why though. From the test run:

```diff
--- Expected
+++ Actual
@@ -13,2 +13,12 @@
 spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: foo
+                operator: In
+                values:
+                  - bar
+                  - baz
   env:
@@ -36,2 +46,7 @@
       memory: 100Mi
+  tolerations:
+    - effect: NoSchedule
+      key: foo
+      operator: Exists
+      value: value1
   ttlSecondsAfterFinished: 100
```

Reply:
Will be fixed in #2008